### PR TITLE
[26426] Move Project table from admin area to global Project list page

### PR DESCRIPTION
**Pull request by HDinger**
Move the project table from the admin area to the global overview page.

#### TODO

- [x] Instead of the current list there is a table shown when clicking on "View all projects".
- [x] The project section in the admin area shall be removed.
- [x] The new table must handle permissions of the user correctly.
- [x] The project description shall be visible in the table (e.g. in an extra row).
- [x] The projects the user is member of shall be marked.
- [x] Show menu on the right instead of three icons.
- [ ] Show info text only for admins
- [ ] There is an icon to show/hide description (visible for all users that may see the description)
- [ ] There is a button that shows/hides all descriptions
- [ ] The more icon is only visible on hover

- [x] Take care that the licenses plugin works as well (requirement: remove filter from project section) https://github.com/opf/openproject-licenses/pull/7

https://community.openproject.com/projects/openproject/work_packages/26426/activity

### CODE REVIEW

**Comment by HDinger** [comment]
#### Current state
bildschirmfoto 2017-09-29 um 16 24 27

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/4dee4f5073567882d2614a2050134ab9d7070930) (deployed commit: aa4454cc885477d2210bd028d510cfe5c2023779).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/305cd36d0c26ac0f107594c5f8771df05ffce965) (deployed commit: aa4454cc885477d2210bd028d510cfe5c2023779).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/e69c29aab86f42dacfd653670a3882a79c42995d) (deployed commit: 365cd6dbac8208bbdc02a9fa7c07278095b87176).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/baaeb2d61ebfa7ddceb083adbb1a9e66c71fbf4b) (deployed commit: d896c03e571daddf6bf880c2376f24eceb74adc0).

**Comment by HDinger** [comment]
@wielinde Can you take on the review please?

**Comment by wielinde** [comment]
Hey Henriette,

I start a list of things, that I come accross, so that I don't forget them:

- [x] I believe that subject and description should link to a project's overview instead of a project's settings page.
- [x] Remove hover for columns (for now) as it is not clear how the description shall be higlighted as it spans multiple columns.
- [x] Align more menu icon vertically (I am actually surprised it is not centred. I thought it was centred within the font)
- [x] Limit width of description (as we started do in other places to improve readability
- [x] Some functionality to expand description. Default should be a collapsed state.

Nice to have:
- [x] (Star icon needs explanation)
- [x] (Move star icon to an own column?)
- [ ] Add a placeholder for the Project filter input field that explains the field (something like "Name contains...")

openproject___projects

File: app/controllers/copy_projects_controller.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def project_copy
   end
 
   def origin
-    params[:coming_from] == 'admin' ? projects_admin_index_path : settings_project_path(@project.id)
+    params[:coming_from] == 'admin' ? projects_path : settings_project_path(@project.id)
```
Comment (wielinde) [inline app/controllers/copy_projects_controller.rb:89]: Is that case actually still possible? Where can an admin still copy a project from within the admin area?

File: spec_legacy/functional/projects_controller_spec.rb [hunk 2]
```diff
@@ -64,14 +64,6 @@
     assert_select('a', { content: /Private child of eCookbook/ }, false)
   end
 
-  it 'should index atom' do
```
Comment (wielinde) [inline spec_legacy/functional/projects_controller_spec.rb:67]: Do I understand it right, that ```/projects.atom``` will not exists in future? If so: how do the links that are stored in peoples feed readers and systems deprecate? I am asking to make sure that everybody is clear on the consequences.

**Comment by HDinger** [comment]
@wielinde I included your feedback.

* As discussed we decided to remove the rss feed possibility.
* After a discussion with Niels we decided to remove the info about projects I am member of. Thus we do not have the problems of a strange icon and a too noisy hierarchy.
* Since we have to change the filter section completely soon I decided not to put any effort in this now. We have a newer implementation of filters which is not applied here yet. Jens is going to have a look at the this.

**Comment by HDinger** [comment]
@wielinde I had a longer discussion with Niels, but we did not really come to a conclusion. I implemented the things he mentioned so that you can have a look at this together tomorrow.

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/916496ffeec8dc3516a71e53d85234391e90779c) (deployed commit: 60fc80eaab1e235ce73e9c305915a78a6c363277).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/0449e63c320525a5fd99cc42450adb053086af7a) (deployed commit: 65f81ac8c2eb652ce7893d5fdd955faf02c9c2aa).

**Comment by HDinger** [comment]
### Current State
bildschirmfoto 2017-10-05 um 19 16 10

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/cf2ab18927c70055b15542d0765da53441f46425) (deployed commit: e2ac947d36df9dd577c222c20469eeaac446e514).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/8ee331a3524da86dd6652b2948bd3a6dfa883da1) (deployed commit: ae1ab2b01c57798178f908f424242ede637a266e).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/e7b56e668ba1d66089c43b7eca298b94db7baec7) (deployed commit: 56cb430ba586564a33a007ae99157a5eae2d2841).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/14a2c7c3879b498322194ef5bfc72b2477c5ddfe) (deployed commit: 7d3687489fa28680a862e9f479f9cd66b5b6a213).

**Pull request merged**